azure-pipelines.yml

```diff
@@ -18,11 +18,11 @@ steps:
     gradleWrapperFile: './OneSignalSDK/gradlew'
     gradleOptions: '-Xmx4096M'
     javaHomeOption: 'JDKVersion'
-    jdkVersionOption: '1.8'
+    jdkVersionOption: '17'
     jdkArchitectureOption: 'x64'
     publishJUnitResults: false
     testResultsFiles: '**/TEST-*.xml'
-    tasks: 'clean build -x :unittest:check'
+    tasks: 'clean build'
```

Comment:
Should we be removing `-x :unittest:check` here?

Reply:
Yes, because as mentioned before this is running the unittest project as an indepent package and because the old structure of the OneSignalSDK is making the build crash

Reply:
Same answer as above: can we then keep `-x :unittest:check` commented so that later we know we have to address it?

Reply:
Yes we can